neqo-http3/src/connection.rs

```diff
@@ -854,7 +854,7 @@ impl Http3Connection {
             (Some(s), Some(_r), false) => {
                 if s.http_stream()
                     .ok_or(Error::InvalidStreamId)?
-                    .send_headers(&[Header::new(":status", "400")], conn)
+                    .send_headers(&[Header::new(":status", "404")], conn)
```

Comment:
No test for this?

Reply:
We currently do not propagate the response code. Therefore it is not possible to test this.

We should propagate the response code. I will submit an update.